frontend/packages/kubevirt-plugin/src/components/virtualization-overview/getting-started-card/RecommendedOperatorsCard.tsx

```diff
@@ -14,23 +14,21 @@ export const RecommendedOperatorsCard: React.FC = () => {
   const moreLink: GettingStartedLink = {
     id: 'openshift-virtualization-recommended-operators',
     title: t('kubevirt-plugin~Learn more about Operators'),
-    href: 'https://www.openshift.com/blog',
+    href: `${window.SERVER_FLAGS.basePath}operatorhub/all-namespaces`,
```

Comment:
```suggestion
    href: `/operatorhub/all-namespaces`,
```
can we make it an internal link ?